### PR TITLE
task/STC-19

### PR DESCRIPTION
**Pull request by jcoro**
I updated the font to Open Sans as indicated (with some minor subsequent changes to header buttons/links).  I still didn't alter the index.html homepage, as another team member claimed it, so the linespacing is still off.  I'm also currently nearing completion on CRUD and filesystem operations for Data Sets and Data Files.  Everything is functional - I just want to look it over before I commit.

### CODE REVIEW

File: scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/LoginController.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package edu.asu.diging.scriptoocloud.web;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class LoginController {
+    @RequestMapping(value = "/login")
+    public String login(Model model) { return "login"; }
```
Comment (jdamerow) [inline scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/web/LoginController.java]: Please follow our code style

**Comment by jdamerow** [comment]
Make it so, Jenkins.

**Review by jdamerow — CHANGES_REQUESTED** [review]
Besides that it should be good to go.

**Comment by jcoro** [comment]
I believe my IDE may automatically format the files in this way if the method consists of a single return statement.  I updated the branch to put the return statement on its own line in the LoginController, but could you clarify - Is it the "one statement per line" rule that's at issue here?  If so, I'll try to adjust my IDE settings.

**Comment by jdamerow** [comment]
Yes. We're following the google code style (but with 4 spaces indentation). There should be a setting in your IDE to adjust that. If you're using Eclipse, you can follow this: https://diging.atlassian.net/wiki/spaces/DIGING/pages/2057011201/Java+Code+Style

**Pull request merged**